db/migrate/20220202122417_add_parent_id_to_miq_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddParentIdToMiqRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :miq_requests, :miq_requests
```

Comment:
I think you want belongs_to (which is add_reference under the covers).  @agrare this is what you were suggesting right?

```suggestion
    add_reference :miq_requests, :parent, :type => :bigint
```

Reply:
>  this is what you were suggesting right?

:+1: yes, exactly

Reply:
Changed